README.md

```diff
@@ -29,89 +29,14 @@ The download consists of 2 parts:
 1. VIB (vDVS driver): The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
 2. Managed plugin (vDVS plugin): Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
 
-Pick the latest release and use the same version of vDVS plugin (on Docker host VM) and driver (on ESX).
-
-## Demos
-
-The demos are located on the project [site](https://vmware.github.io/docker-volume-vsphere/documentation) and [wiki](https://github.com/vmware/docker-volume-vsphere/wiki/Demos)
-
-## Project Website
-
-Project page is located @ [https://vmware.github.io/docker-volume-vsphere/](https://vmware.github.io/docker-volume-vsphere). Documentation, FAQ and other content can be found @ [https://vmware.github.io/docker-volume-vsphere/documentation](https://vmware.github.io/docker-volume-vsphere/documentation)
+Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) for vDVS driver's & plugin's installation instruction. Pick the latest release and use the same version of vDVS plugin (on Docker host VM) and driver (on ESX).
```

Comment:
Please reword the text as follows - "for installation instructions for vDVS driver and managed plugin.
Download the release from <bintray location>. To ensure compatibility, make sure to use the same version of the vDVS driver and Managed plugin."